### DISCUSSION TITLE
"Used by" links in field settings sidebar should link to global set's settings

### DISCUSSION THREAD

**Discussion by BigglesZX** [craftcms/cms#15423] [Ideas] [answered]
Just a little thing I noticed this morning. When viewing a field's settings, there's a helpful list of "Used by" links in the sidebar where entry types and global sets that use the field are shown. For an entry type, the link points to the settings page for that type. However for a global set, the link points to the content edit view for the global set, rather than its settings page. It would be helpful if the behaviour for global sets was the same as entry types.

A PR for this ought to be within my capabilities, if that would be helpful?

<img width="1160" alt="Screenshot 2024-07-26 at 11 40 59" src="https://github.com/user-attachments/assets/dc3a9783-0c63-45a8-9745-ce9a9e7e5303">

**Comment by brandonkelly** [comment, marked as answer]
Good call. Just made that the case for Craft 5.3 (13b722ea7c669bbee00a154b27a608e7f974df92).
